tests/client/test_client_multi_search_meilisearch.py

```diff
@@ -32,3 +32,28 @@ def test_multi_search_on_no_index(client):
     """Tests multi-search on a non existing index."""
     with pytest.raises(MeilisearchApiError):
         client.multi_search([{"indexUid": "indexDoesNotExist", "q": ""}])
+
+
+def test_multi_search_with_no_value_in_federation(client, empty_index):
+    """Tests multi-search with federation, but no value"""
+    empty_index("indexA")
+    response = client.multi_search([{"indexUid": "indexA", "q": ""}], {})
+
+    assert response["hits"][0]["_federation"]['indexUid'] == "indexA" if len(response["hits"]) >= 1 else isinstance(response, dict)
+
+
+def test_multi_search_with_offset_and_limit_in_federation(client, empty_index):
+    """Tests multi-search with federation, with offset and limit value"""
+    empty_index("indexA")
+    response = client.multi_search([{"indexUid": "indexA", "q": ""}], {"offset": 2, "limit": 2})
+    
+    assert len(response["hits"]) == 2 if response['hits'] else True
+
+
+def test_multi_search_with_federation_options(client, empty_index):
+    """Tests multi-search with federation, with federation options"""
+    empty_index("indexA")
+    response = client.multi_search([{"indexUid": "indexA", "q": "", "federationOptions": {"weight": 0.99}}], {"offset": 2, "limit": 2})
+
+    assert isinstance(response["hits"], list)
+    assert response["hits"][0]["_federation"]["weightedRankingScore"] < 0.99 if len(response["hits"]) >= 1 else True
```

Comment:
These tests should be more specific. There shouldn't be if/else in the asserts because we know what the results should be. Let's also use an index that returns results so we can verify `_federation` is present. For example `test_multi_search_with_no_value_in_federation` could be something like:

```py
from tests.common import INDEX_UID


def test_multi_search_with_no_value_in_federation(client, empty_index, index_with_documents):
    """Tests multi-search with federation, but no value"""
    index_with_documents()
    empty_index("indexB")
    response = client.multi_search(
        [{"indexUid": INDEX_UID, "q": ""}, {"indexUid": "indexB", "q": ""}], {}
    )

    assert "results" not in response
    assert len(response["hits"]) > 0
    assert "_federation" in response["hits"][0]
    assert response["limit"] == 20
    assert response["offset"] == 0
```

Reply:
Makes sense, will do that. Thank you.